### ISSUE TITLE
connection issue on debian 

### ISSUE THREAD

**Issue by Behnamgh** [microsoft/azuredatastudio#9210]
- Azure Data Studio Version:

Steps to Reproduce:

1. I can't connect to SQL database, with Debian 10 it doesn't work. It didn't show any error but keep working without any response.
By entering the wrong URL it shows error

**Comment by kburtram** [comment]
@Behnamgh do you know if there are any errors in "Help->Toggle Developer Tools" Console log?

**Comment by Behnamgh** [comment]
Today I stated to install again SQL server plugin for visaul studio code. during installation I saw some missing downloading file in:
 /home/<user>/.vscode/extensions/ms-mssql.mssql-1.8.0/sqltoolsservice/1.8.0/.

The missing file was: 
https://download.microsoft.com/download/e/7/8/e781cf8f-9c3f-4ad5-bd0c-f3c62b8bc6bd/microsoft.sqltools.servicelayer-rhel-x64-netcoreapp2.2.tar.gz

I downloaded this file and copy it into the target directory then install it.
it solved all my problems(both plugin and azure data studio).

By the way, Thank you very much for your time
